### PR TITLE
Nfdiv 1676

### PR DESCRIPTION
**Pull request by velimd**
### JIRA link ###
https://tools.hmcts.net/jira/browse/NFDIV-1676

### Change description ###
image
image

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/test/functional/features/sole-conditional-order.feature [hunk 1]
```diff
@@ -52,21 +52,6 @@ Feature: Sole conditional order
...
-    Then the page should include "Upload any documents"
-
-    When I clear the form
-    Given I select "If the court wants you to explain something or provide additional information then write your response here. If the court has just asked you to upload documents then you do not have to write anything, unless you think it’s useful information."
-    And I type "test details"
-    And I select "I cannot upload some or all of my documents"
-    Then the page should include "You can post your documents to the court"
-
-    When I click "Continue"
-    Then the page URL should be "/hub-page"
```
Comment (lara-mpt) [inline src/test/functional/features/sole-conditional-order.feature:68]: how comes you deleted this?

File: src/main/steps/applicant1/hub-page/sole/content.ts [hunk 2]
```diff
@@ -186,6 +180,43 @@ const en = ({ isDivorce, partner, userCase }: CommonContent) => ({
...
   },
+  serviceApplicationRejected: {
+    line1: {
+      part1: `The court has refused your application ${
+        alternativeServiceType === AlternativeServiceType.BAILIFF
+          ? 'for bailiff service'
+          : alternativeServiceType === AlternativeServiceType.DEEMED
+          ? 'for deemed service'
+          : 'to dispense with service'
+      }. You can read the reasons on the court’s `,
```
Comment (lara-mpt) [inline src/main/steps/applicant1/hub-page/sole/content.ts:191]:
```suggestion
      part1: `The court has refused your application ${
        alternativeServiceType === AlternativeServiceType.BAILIFF
          ? 'for bailiff'
          : alternativeServiceType === AlternativeServiceType.DEEMED
          ? 'for deemed'
          : 'to dispense with'
      } service. You can read the reasons on the court’s `,
```

**Review by lara-mpt — APPROVED** [review]

File: src/test/functional/features/sole-conditional-order.feature (hunk 1 above)
Comment (velimd) [reply src/test/functional/features/sole-conditional-order.feature:68]: this test is repeated on the sole-hub-page.feature

Comment (velimd) [reply src/test/functional/features/sole-conditional-order.feature:68]: if you think its worth having it here and deleting it in the hub page one then i can go for that but i dont think its worth having both.

**Pull request merged**